lib/wifi_user/repository/user.rb

```diff
@@ -26,7 +26,7 @@ def random_username
   end
 
   def generate_username
-    ALPHABET_WITHOUT_VOWELS.sample(6).join
+    (0...6).map { ALPHABET_WITHOUT_VOWELS[rand(6)] }.join
```

Comment:
Will this ever touch letters beyond "h"?
This 6 looks like it's going to only use a small subset of our alphabet.

Reply:
Oh yeh my bad!!